rest-api-guidelines/core-principles/testing.md

```diff
@@ -1,6 +1,18 @@
 # Testing
 
+## DREED
```

Comment:
Typo? "Dredd"?

Reply:
Ups. Sorry. Thanks! Corrected in new pull request.